telemetry-http-okhttp/README.md

```diff
@@ -1,4 +1,4 @@
-This module contains reference implementations of the required interfaces for the SDK to function.
+This module contains reference implementations of the required http client for the SDK to function.
```

Comment:
nitpick: should be singular `a reference implementation`

Reply:
:ack:

Reply:
check the changes